### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Khabailiuk**
- [DEMO LINK](https://Khabailiuk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Khabailiuk.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Comment by Khabailiuk** [comment]
- [DEMO LINK](https://Khabailiuk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Khabailiuk.github.io/layout_moyo-header/report/html_report/)

**Comment by TarasHoliuk** [comment]
Pay attention to code style:
image

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="logo.png" alt="logo">
+      </a>
+      <nav class="nav">
```
Comment (TarasHoliuk) [inline src/index.html:16]: what have you deployed?
image

it doesn't match your code. please, deploy the latest version

File: src/style.css [hunk 2]
```diff
@@ -1 +1,34 @@
...
+.nav__list {
+display: flex;
+flex-wrap: wrap;
+list-style: none;
+margin: 0;
+padding: 0
+}
+.nav__link{
+display: block;
+line-height: 60px;
```
Comment (TarasHoliuk) [inline src/style.css:19]: it's better to add padding instead

File: src/style.css [hunk 3]
```diff
@@ -1 +1,34 @@
...
+}
+.nav__list {
+display: flex;
+flex-wrap: wrap;
+list-style: none;
+margin: 0;
+padding: 0
+}
+.nav__link{
+display: block;
```
Comment (TarasHoliuk) [inline src/style.css:18]: remove it

File: src/style.css [hunk 4]
```diff
@@ -1 +1,34 @@
...
+line-height: 60px;
+text-decoration: none;
+font-style: normal;
```
Comment (TarasHoliuk) [inline src/style.css:21]: redundant

File: src/style.css [hunk 5]
```diff
@@ -1 +1,34 @@
...
+}
+
+
+.logo{
+width:40px;
+height:40px
+}
+.nav__list {
+display: flex;
+flex-wrap: wrap;
```
Comment (TarasHoliuk) [inline src/style.css:12]: why do you need it?

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Fix TEST REPORT LINK

**Pull request closed without merging**